lib/rouge/lexers/console.rb

```diff
@@ -179,7 +179,7 @@ def prompt_regex
       end
 
       def stream_tokens(input, &output)
-        input = StringScanner.new(input)
+        input = Rouge.string_scanner(input)
```

Comment:
I abstracted this into a method to prevent having to check all the time what the supported mode is.